spec/rspec/core/source/syntax_highlighter_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 class RSpec::Core::Source
   RSpec.describe SyntaxHighlighter do
-    let(:config)      { RSpec::Core::Configuration.new.tap { |c| c.color = true } }
+    let(:config)      { RSpec::Core::Configuration.new.tap { |config| config.no_color = false } }
```

Comment:
Thanks for the explanation.  I mis-understood how you were setting `no_color = true` in `spec_helper.rb` -- I didn't realize you were monkey patching (or updating and existing monkey patch) to make _all_ instances of `RSpec::Core::Configuration` in our spec suite default to `no_color = true`.

Instead, can you move `config.no_color = true` to the `sandboxed` block in [spec/support/sandboxing.rb](https://github.com/rspec/rspec-core/blob/6a4376e276e52b89d35aa5e16b74758e9f64b991/spec/support/sandboxing.rb#L7)?  The sandboxing ensures each spec runs with a new, fresh instance of config to prevent state leaks when particular specs modify config--so if you set it there, it'll ensure `no_color = true` is set for all specs in our suite by default w/o the need to monkey patch _all_ instances of config, and then I believe you can remove this `tap` block here.